cmd/fluxctl/release_cmd.go

```diff
@@ -160,7 +159,7 @@ func (opts *controllerReleaseOpts) RunE(cmd *cobra.Command, args []string) error
 			return err
 		}
 
-		spec, err := promptSpec(cmd.OutOrStdout(), result, opts.verbosity)
+		spec, err := promptSpec(cmd.OutOrStdout(), result, opts.verbosity, opts.force)
```

Comment:
Given that the `force` argument just gets written into the spec, you could do that here after calling an unmodified `promptSpec`.

But I wonder if this is the most useful interpretation of "force" in the interactive mode. To use it, you have to decide ahead of time whether you need it -- and you don't necessarily know what would be left out if not used, so you have to figure that out first. Interactively, you might expect to be shown updates that would occur _except that_ they are locked, and select them individually. Not a requirement for this PR.